packages/blockchain-link-types/src/blockbook.ts

```diff
@@ -153,6 +153,7 @@ export interface Transaction {
         name: string;
         symbol: string;
         decimals?: number;
+        type: 'ERC20' | 'ERC1155' | 'ERC721';
```

Comment:
`type` is a name from `blockbook`. 

we have `type` occupied by `sent`, `recv`,.. so i called it standard. I agree that it would be nice to unify it, but it would be a looot of changes